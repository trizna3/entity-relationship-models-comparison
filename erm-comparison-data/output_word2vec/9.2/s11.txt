boards-board
weather-weather
fuel type-weather;board
tile-tile
games-game
tile type-coordinates
units-units
MOVE_ATTR_TO_INCIDENT_ASSOCIATION:S-points-(units(MANY),fuel(ONE))
REBIND_MN_TO_1NN1:S-board-tile
REBIND_MN_TO_1NN1:S-weather-board
