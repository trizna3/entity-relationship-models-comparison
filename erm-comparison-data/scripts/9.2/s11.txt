entity set game {
};

entity set board {
	"rows"
	"columns"
};

entity set tile {
	"swamp"
	"field"
	"forest"
	"mountain"
};

entity set weather {
	"rainy"
	"sunny"
	"cold"
};

entity set units {
	"name"
	"health"
	"attack"
	"defense"
	"points"
};

entity set movement {
	"points"
};

entity set fuel {
	"feet"
	"horse"
	"belts"
};

association between 
{
	game ONE
	board ONE
}{};

association "coordinates" between 
{
	board MANY
	tile MANY
}{};

association between 
{
	tile ONE
	units ONE
}{};

association between 
{
	units MANY
	fuel ONE
}{};

association between 
{
	weather MANY
	board MANY
}{};