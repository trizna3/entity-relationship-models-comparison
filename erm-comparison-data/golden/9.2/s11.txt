games-game
boards-board
tile-coordinates
tile type-tile
weather-weather
fuel type-fuel
units-units
move-movement
REBIND_NARY_ASSOCIATION:E-weather;fuel type;tile type
REBIND_MN_TO_1NN1:S-board-tile